spec/presenters/aaib_report_presenter_spec.rb

```diff
@@ -35,24 +35,59 @@
     }
   }
 
-  let(:schema) { double(:schema) }
+  let(:finder) { OpenStruct.new(
+    :base_path => "/aaib-reports",
+    :details => OpenStruct.new(
+      :beta => false,
+      :beta_message => nil,
+      :facets => [
+        OpenStruct.new(
+          :key => "aircraft_category",
+          :name => "Aircraft category",
+          :allowed_values => [
+            OpenStruct.new(
+              :label => "Big Aeroplanes",
+              :value => "big-aeroplanes"
+            )
+          ]
+        ),
+        OpenStruct.new(
+          :key => "report_type",
+          :name => "Report type",
+          :allowed_values => [
+            OpenStruct.new(
+              :label => "a report type",
+              :value => "a-report-type"
+            )
+          ]
+        ),
+      ]
+    )
+  ) }
 
-  let(:schema_response) {
+  let(:finder_response) {
     {
-      "aircraft_category" => {
-        label: "Aircraft category",
-        values: [
-          {label: 'Big Aeroplanes', slug: "big-aeroplanes"}
+      "market_sector" => {
```

Comment:
Should this be `aircraft_category` for consistency with the rest of the data in the spec?